### ISSUE TITLE
rss feed not recognized by liferea

### ISSUE THREAD

**Issue by ulrichard** [miguelfreitas/twister-core#308]
Trying to subscribe to http://127.0.0.1:28332/rss?account=ulrichard with liferead leads to the following error (sorry for the German):

Das letzte Aktualisieren dieses Abonnements schlug fehl!
HTTP-Fehlercode 503: Server-Fehler
Es traten Fehler beim Parsen des Feeds auf!
Details
Konnte Feedtyp nicht bestimmen! Bitte überprüfen Sie, ob die Adresse wirklich auf eine Dokument in einem der unterstützten Formate zeigt!

XML Parser Ausgabe:
Die Adresse die Liferea abonnieren soll zeigt auf eine Webseite und die automatische Feed-Erkennung hat keine Feeds gefunden. Möglicherweise unterstützt die Webseite die automatische Feed-Erkennung nicht.Quelle zeigt auf ein HTML-Dokument.

Kontaktieren Sie den Autor/Webmaster dieses Feeds zu diesem Problem!

Basically, it complains, that the served document is a website rather than a feed.
If I open the address in Firefox, it looks fine, like a regular feed.

**Comment by digital-dreamer** [comment]
OK, I'm debugging this now. I have Liferea installed on Debian, and it works just fine for me. We have to find out what's wrong. Can you please try to put just http://127.0.0.1:28332/rss (without the account specifier) into Liferea? Does the error message go away?

**Comment by ulrichard** [comment]
When I ran it with full debug output, I discovered that it's a liferea issue. 
I have it configured to usually use a proxy. Eventhough I configured it to not use the proxy for this particular feed, it still tries to fetch it through the proxy. 
When I configure the following command instead of directly fetching an URL in liferea, it does work:
wget --no-proxy http://user:pass@127.0.0.1:28332/rss && cat rss

But now I have another problem: The feed contains only posts from @twister, and not from all the other users that I follow and see on the postboard. This is the same wether I watch the rss feed with liferea or in the browser.

**Comment by digital-dreamer** [comment]
I think it is something different. The “account=” parameter specifies the account that you want to log in and get the feed from. When the account parameter is not used, twister just picks the first account. So, if you enter just /rss, it loads the feed from your other account, where you don't follow anyone, except @twister, which is the default subscription.

And if /rss works, it explains the first problem, where Liferea says that the feed is a web page. When you open the feed in a web browser, and put a wrong username in the account=, it will display an error message instead of loading the RSS feed. When you do the same with Liferea, it will only see that the returned content (the plaintext error message) does not contain RSS tags, and say it is not a feed.

**Comment by ulrichard** [comment]
The problem was, that the file already existed, and was not overwritten. The following command is better:
wget --no-proxy -q --output-document - http://user:pass@127.0.0.1:28332/rss?account=ulrichard

The 503 page before was the error page from the proxy.

So after all, nothing wrong with twister. But thanks for the help anyway.
